dev/com.ibm.ws.security.mp.jwt/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -57,6 +57,9 @@ audiences.desc=The trusted audience list to be included in the aud claim in the
 clockSkew=The time difference allowed between systems
 clockSkew.desc=This is used to specify the allowed clock skew in minutes when validating the JSON web token.
```

Comment:
Not sure if you can change this at this time but if so:

This is used to specify the allowed clock skew in minutes when validating the JSON web token.
--->
This attribute specifies the allowed clock skew in minutes when the JSON web token is validated.